libraries/cli/include/cli/default_config.hpp

```diff
@@ -166,6 +166,7 @@ const char *default_json = R"foo({
     },
     "pbft": {
       "committee_size": "0x3e8",
+      "number_of_proposers": "0x14",
```

Comment:
from my perspective as someone who is just reviewing this PR, I did not know that `number_of_proposers` is basically a specific committee size for proposal round. Wouldn't it be more intuitive if we call it something like `propose_round_committee_size`, `proposing_committee_size` or some name that would also include committe_size in it ?